<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create</class>
 <widget class="QWidget" name="create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(20, 181, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="signupL">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="usernameRF">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string>Enter Your Name </string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::VisualMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailRF">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter Your  Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwRF">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Enter Your Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwcRF">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password Confirm </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="profboxR">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Volunteer</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Volunteer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Consumer</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="unR">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
        </property>
        <property name="text">
         <string>User Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emailR">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pwR">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pwcR">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
        </property>
        <property name="text">
         <string>Password Confirm </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="profR">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="back2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="signupR">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Press To Signup</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="error">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
